lighthouse/tests/exec.rs

```diff
@@ -23,19 +24,22 @@ pub trait CommandLineTestExec {
     /// Executes the `Command` returned by `Self::cmd_mut` with temporary data directory, dumps
     /// the configuration and shuts down immediately.
     ///
-    /// Options `--datadir`, `--dump-config` and `--immediate-shutdown` must not be set on the
-    /// command.
+    /// Options `--datadir`, `--dump-config`, `--dump-chain-config` and `--immediate-shutdown` must
+    /// not be set on the command.
     fn run(&mut self) -> CompletedTest<Self::Config> {
         // Setup temp directory.
         let tmp_dir = TempDir::new().expect("Unable to create temporary directory");
         let tmp_config_path: PathBuf = tmp_dir.path().join("config.json");
+        let tmp_chain_config_path: PathBuf = tmp_dir.path().join("chain_spec.yaml");
 
-        // Add args --datadir <tmp_dir> --dump-config <tmp_config_path> --immediate-shutdown
+        // Add args --datadir <tmp_dir> --dump-config <tmp_config_path> --dump-chain-config <tmp_chain_config_path> --immediate-shutdown
         let cmd = self.cmd_mut();
         cmd.arg("--datadir")
             .arg(tmp_dir.path().as_os_str())
-            .arg(format!("--{}", "--dump-config"))
+            .arg(format!("--{}", "dump-config"))
```

Comment:
Yikes good catch! Funny that the old code still worked as intended :sweat_smile: